common/shlibs

```diff
@@ -3401,3 +3401,10 @@ libbelcard.so.1 belcard-1.0.2_1
 libshaderc_shared.so shaderc-2017.2_1
 libglslang.so glslang-6.2.2596_1
 libSPIRV.so glslang-6.2.2596_1
+libsecp256k1.so.0 libsecp256k1-20180531_1
+libbitcoin.so.0 libbitcoin-system-3.5.0_1
```

Comment:
are all those required ? if one isn't immediately required by a package don't add to common/shlibs

Reply:
Yes, all are required.